src/ui/component/stepper/utils.ts

```diff
@@ -0,0 +1,26 @@
+import type {
+  ProgressBarColor,
+  ProgressBarTransition
+} from './stepperProgress/progressBar/progressBar'
+import type { StepStatus } from './stepperProgress/stepperProgress'
+
+export const getProgressBarColor = ({ valid, active }: StepStatus['status']): ProgressBarColor =>
+  active ? 'blue' : valid ? 'green' : 'grey'
+
+export const getProgressBarTransition = ({
```

Comment:
If I understand well, these methods are only used in `StepperProgress` component, so I would have let them in `StepperProgress.tsx` as they are not shared

Reply:
The purpose was to isolate types that are not directly related to the progress bar, e. g. `StepStatus`. We can definitely move it in into the progressBar file if this doesn't bring that much value.

Reply:
The purpose was to isolate types that are not directly related to the progress bar, e. g. `StepStatus`. We can definitely move it in into the progressBar file if this doesn't bring that much value.